spec/Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: all spec idl clean
```

Comment:
should you update the toplevel makefile to also build this?

Reply:
yep, i'll kick off another PR so that i can make this PR clean.